src/generateResponseType.js

```diff
@@ -66,6 +67,13 @@ const sortedObjectTypeAnnotation = (
         undefined /* internalSlots */,
         true /* exact */,
     );
+    const name = config.path.join('_');
+    if (config.allObjectTypes != null && config.path.length > 1) {
+        config.allObjectTypes[name] = obj;
+        return babelTypes.genericTypeAnnotation(babelTypes.identifier(name));
+    } else {
+        return obj;
+    }
```

Comment:
What's the `config.path.length > 1` check for?

Reply:
Hm I could refactor this to make more sense. So the first item in `config.path` is the name of the operation (e.g. `SomeQuery_`), if that's the only item, then this is the top level, and we're not at the "sub-object" level yet.